src/algebra/order/field/power.lean

```diff
@@ -143,6 +143,8 @@ alias even.zpow_pos_iff ↔ _ even.zpow_pos
 alias odd.zpow_neg_iff ↔ _ odd.zpow_neg
 alias odd.zpow_nonpos_iff ↔ _ odd.zpow_nonpos
 
+@[simp] lemma abs_zpow (a : α) (n : ℤ) : |a ^ n| = |a| ^ n := map_zpow₀ (abs_hom : α →*₀ α) a n
```

Comment:
This feels like a weird place for this lemma; do we have `abs_inv` and `abs_div` already that it could go next to?

Reply:
We do have the lemmas in `algebra.order.field.basic`, but `map_zpow₀` isn't available there. Also note that `algebra.order.field.power` is precisely about `zpow` and nothing else.

Reply:
Ah, in which case I'll reduce my comment to "sure;y this doesn't belong in this PR"!

Reply:
Will swallow it up with a few other isolated lemmas once #18607 and #18609 are in.